extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/util/MultiStageQueryContext.java

```diff
@@ -58,10 +58,14 @@
   public static final String CTX_FINALIZE_AGGREGATIONS = "finalizeAggregations";
   private static final boolean DEFAULT_FINALIZE_AGGREGATIONS = true;
 
-  public static final String CTX_ENABLE_DURABLE_SHUFFLE_STORAGE = "durableShuffleStorage";
+  public static final String CTX_DURABLE_SHUFFLE_STORAGE = "durableShuffleStorage";
```

Comment:
I have featured flagged it just in case we break something. In the future, yes we should remove the durableShuffleStorage context param. 

The way it works is 

if the user sets faultTolerance=true, I enable durableShuffleStorage as well 
if the user sets faultTolerance=true, and durableStorage=false its an error. 
if the user sets durableStorage=true, faultTolerance context param remains untouched.